### ISSUE TITLE
Using WxsFiles and OutDir causes build to fail

### ISSUE THREAD

**Issue by pmfws** [oleg-shilo/wixsharp#96]
I have a wix fragment file that i want to use in my installer but when I add it to the Project.WxsFiles list I cannot longer use non-default Project.OutDir.
Simplified Setup.cs source:
```c#
        Project project =
                new Project("OutDirTest",
                    new Dir(@"%ProgramFiles%\OutDirTest",
                        new ExeFileShortcut("Uninstall OutDir Test",
                                            "[System64Folder]msiexec.exe",
                                            "/x [ProductCode]")));

        project.WxsFiles.Add(@"MyFragment.wxs");
        project.UI = WUI.WixUI_Minimal;
        project.SourceBaseDir = Environment.CurrentDirectory;
        project.OutDir = @"Res\Setup";
        project.OutFileName = "Setup";

        Compiler.BuildMsi(project);
```
Fails with
> The system cannot find the file 'MyFragment.wixobj' with type 'Source'.

**Comment by oleg-shilo** [comment]
Can you please provide the MyFragment.wxs file content. Initially I thought that Wix# cannot find it bu it seems like some other problem...

**Comment by oleg-shilo** [comment]
Don't worry I managed to reproduce it now...

**Comment by oleg-shilo** [comment]
Fixed in the latest release (v1.4.6.2). Use NuGet to get the fix.
